lib/zstd.h

```diff
@@ -1085,6 +1085,17 @@ typedef enum {
 *  Frame size functions
 ***************************************/
 
+/**
+ * Contains the compressed frame size and an upper-bound for the decompressed frame size.
+ * Note: before using `compressedSize` you must check for errors using ZSTD_isError().
+ *       similarly, before using `decompressedBound`, you must check for errors using:
+ *          `decompressedBound` != ZSTD_CONTENTSIZE_ERROR
+ */
+typedef struct {
+    size_t compressedSize;
+    unsigned long long decompressedBound;
+} ZSTD_frameSizeInfo;
```

Comment:
This doesn't belong in `zstd.h`, since it is a public header, and this is not a public function.  You can put it in `zstd_internal.h` and include that in `zstd_legacy.h`.

Reply:
I was wondering how to get around this. Thanks!